lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelResult.cxx

```diff
@@ -284,6 +298,31 @@ Point LinearModelResult::getCoefficientsStandardErrors() const
   return standardErrors;
 }
 
+LeastSquaresMethod LinearModelResult::buildMethod() const
+{
+  LeastSquaresMethod leastSquaresMethod;
+  leastSquaresMethod = LeastSquaresMethod::Build(ResourceMap::GetAsString("LinearModelAlgorithm-DecompositionMethod"), design_);
+  const UnsignedInteger basisSize =  design_.getNbColumns();
+  Indices addedIndices(basisSize);
+  addedIndices.fill();
+  Indices conservedIndices(0);
```

Comment:
From my point of view, the current is semantically correct:

- With `addedIndices.fill()`:
DBG - addedIndices = [0,1,2]
- Without `addedIndices.fill()`:
DBG - addedIndices = [0,0,0]
Furthermore, an exception is generated by `DesignProxy`:
```
what():  Requested indices exceed basis size (3)
```

What I want to do at line 310 is to update the decomposition so that it is initialized. That is, I want to add all indices, conserve none and remove none. 

My conclusion is that this code is correct.